libsolidity/AST.h

```diff
@@ -1112,13 +1112,20 @@ class ElementaryTypeNameExpression: public PrimaryExpression
 };
 
 /**
- * A literal string or number. @see Type::literalToBigEndian is used to actually parse its value.
+ * A literal string or number. @see ExpressionCompiler::endVisit() is used to actually parse its value.
  */
 class Literal: public PrimaryExpression
 {
 public:
-	Literal(Location const& _location, Token::Value _token, ASTPointer<ASTString> const& _value):
-		PrimaryExpression(_location), m_token(_token), m_value(_value) {}
+	enum class ethSubDenomination {
```

Comment:
`EthSubDenomination` or just `SubDenomination`, since we're only dealing with ethereum currency anyway.